SEC/Equities/EquityInstruments.rdf

```diff
@@ -786,6 +786,21 @@
 		<skos:definition>preferred share whose dividends are set, typically according to the terms of the prospectus</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-eq-eq;PricePerShare">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-ip;SecurityPrice"/>
```

Comment:
We already have subclasses of SecurityPrice such as BidPrice, ClosingPrice. I assume that these would need instead to be subclasses of PricePerShare.

Reply:
@rivettp No, because a bid or closing price could also apply to a bond, but if you are creating an individual that is specifically a price per share and a closing price, the individual should be multiply classified by both.